client/src/plugins/user-journey-statistics/event-handlers/DeploymentEventHandler.js

```diff
@@ -0,0 +1,124 @@
+/**
```

Comment:
Do we already track to update our documentation regarding this tracking? I can't find something related in the e[pic issue](https://github.com/camunda/camunda-modeler/issues/2934).

Reply:
It's actually being tracked here: https://github.com/bpmn-io/internal-docs/issues/504

Reply:
Oh that's the epic, nice 🙂 Thanks!